### ISSUE TITLE
Sound notification for when it's your turn to act

### ISSUE THREAD

**Issue by erichanson** [pokerth/pokerth#361]
Would love to have a sound for when it is the user's turn to act.  This is the very most important sound to have, as it tells you when to pay attention if you happen to be looking elsewhere.

**Comment by q4z1** [comment]
Hi erichanson,

there is a sound notification for the player when a period of time (few seconds) is left to make a move.
It should be what you are looking for - although it does not trigger immedeately when it's your turn.

Regards.

**Comment by erichanson** [comment]
Yes, it is really not enough time to click into the small box and type a bet when it's only two seconds.  Literally the most important notification is being alerted when it is your turn to act so you have the time to make a decision and type in a bet.  Without this notification, you are not alerted when your turn starts.

**Comment by doug86i** [comment]
Perhaps the client could check to see if its window has focus and only play the "It's your turn" sound if it detects the window is in the background. Might be annoying to have the sound play all the time.

**Comment by q4z1** [comment]
No plans on that. If you play poker without focus - your problem.
